be/src/exec/hdfs_scanner_text.cpp

```diff
@@ -457,6 +458,20 @@ Status HdfsTextScanner::_create_or_reinit_reader() {
     return Status::OK();
 }
 
+Status HdfsTextScanner::_setup_io_ranges() const {
+    if (_shared_buffered_input_stream != nullptr) {
+        std::vector<io::SharedBufferedInputStream::IORange> ranges{};
+        for (int64_t offset = 0; offset < _scanner_params.file_size;) {
+            const int64_t remain_length =
+                    std::min(static_cast<int64_t>(config::text_io_range_size), _scanner_params.file_size - offset);
+            ranges.emplace_back((io::SharedBufferedInputStream::IORange{.offset = offset, .size = remain_length}));
```

Comment:
If you try to construct an element in-place at the end of vector by `emplace_back`, you need pass the constructor arguments of the element, rather than an element instance.

Reply:
clang-tidy using gcc-10 will compile failed.